src/jupyterlabcontroller/storage/k8s.py

```diff
@@ -110,6 +112,9 @@ def __init__(
         self._method_map = KubernetesKindMethodMapper()
         self._fill_method_map()
 
+        self._config_map = ConfigMapStorage(self.k8s_api, logger)
```

Comment:
All of the k8s wrapping is basically to glue our logging and exceptions to the underlying k8s objects?

Reply:
Yes, having read to the end, it is.